Data/BattleForAzeroth/module.lua

```diff
@@ -7,7 +7,7 @@ function module:OnInitialize()
 		[120899] = {name="Kul'krazahn",locations={[862]={59605640},},quest=48333,},
 		[121242] = {name="Glompmaw",locations={[863]={68405700},},quest=50361,},
 		[122004] = {name="Umbra'jin",locations={[862]={71003240},},},
-		[122639] = {name="Old R'gal",locations={[862]={57804880},},},
+		[122639] = {name="Old R'gal",faction="Alliance",locations={[862]={57804880},},},
```

Comment:
Should be `faction="Horde"` if this is an Alliance-only mob. (It's flipped from my HandyNotes plugins, mostly for legacy reasons.)